src/engine/DebugOverlays.EntityLabel.cs

```diff
@@ -22,6 +23,49 @@ private bool ShowEntityLabels
         }
     }
 
+    private void UpdateLabelColour(IEntity iEntity, Spatial entity, Label label)
+    {
+        if (!iEntity.AliveMarker.Alive)
+        {
+            label.Set("custom_colors/font_color", new Color(1.0f, 0.3f, 0.3f));
```

Comment:
Prefer using the explicit `AddColorOverride` as it's most often used in the codebase.